types/jsreport-chrome-pdf/jsreport-chrome-pdf-tests.ts

```diff
@@ -10,7 +10,7 @@ const jsreport = JsReport({
 });
 
 jsreport.beforeRenderListeners.add('test', (req, res) => {
-    console.log('input', req.template.content);
+    console.log('input', (req.template as JsReport.Template).content);
```

Comment:
I don't think it's a good idea to use `as` here.
Is there a better way?
I really do not have much time on this.

Reply:
reading through some of the other extensions it appears that even when it's not explicitly defined it is still required that the content field have data.

https://github.com/jsreport/jsreport-chrome-pdf/blob/7b5ea5b14d14219f37ad254f4e879550732701ed/lib/chrome.js#L18
https://github.com/jsreport/jsreport-docx/blob/b50fad34b38f55b01eaa9093aa1192749b6ca21f/lib/docx.js#L27

Reply:
The `template.content` is required when you use just `jsreport-core` with lets say `jsreport-handlebars` extension.  However in typical cases, when you use more extensions, the `template.content` gets optional. 

Two examples
`jsreport-template` - lets the caller specify `template.name | shortid` and replace the input `template` with entity from db, so the `template.content` isn't required
`jsreport-docx` - loads the `template.content` from the linked word asset, so the `template.content` isn't required

And there are other examples. 

Thank you for your work here. Don't hesitate to mention me here if you have questions.